ReadMe.md.txt

```diff
@@ -0,0 +1,6 @@
+## 21/12/2020
+	*Calculator UI 
```

Comment:
the .txt at the file is wrong. The extension should be Readme.md

Reply:
Solved!